packages/storage/src/chrome/local.ts

```diff
@@ -11,7 +11,7 @@ declare const MINIFRONT_URL: string;
 
 export const localDefaults: LocalStorageState = {
   wallets: [],
-  fullSyncHeight: 0,
+  fullSyncHeight: undefined,
```

Comment:
This is sort of a bug somewhat relating to: https://github.com/penumbra-zone/web/pull/882.

0 sync height indicates that we've synced the genesis block. So `undefined` should be used as the proper default indicating no sync has been done. Lots of changes in components flow from this one change.